docs/explainer.md

```diff
@@ -286,10 +289,11 @@ session.
         requestId,
         amount,
         assetCode,
-        assetScale
+        assetScale,
+        receipt
       } = event.detail
       if (
-        isValidPayment(paymentPointer, requestId, amount, assetCode, assetScale)
+        isValidPayment(paymentPointer, requestId, amount, assetCode, assetScale, receipt)
```

Comment:
I wonder if this example could be a little confusing by bringing in all these fields; technically payment pointer + receipt should be enough to verify payment

Reply:
I think I kept the `amount` since it's subsequently used in the call to `suspendAdvertising`. Presumably `isValidPayment` verifies both the receipt and that `amount` matches the receipt amount.
Alternatively, receipt could just be passed to `suspendAdvertising`, or `isValidPayment` could return an amount (from the receipt).